content/python/concepts/classes/classes.md

```diff
@@ -71,42 +78,36 @@ blue_home = Home()
 blue_home.paint_wall("blue")
 ```
 
-## Class Attributes
+#### Static Methods
 
-Class attributes are variables that are defined outside of all methods and have the same value for every instance of the class.
+It is also possible to create method that only apply to the class itself, not instances of the class. These can be distinguised with `@classmethod` and `@staticmethod`:
 
 ```py
-class Bird:
-  # Class attribute
-  leg_count = 2
-
-parakeet = Bird()
-parrot = Bird()
-
-print(parakeet.leg_count) # Output: 2
-print(parrot.leg_count)   # Output: 2
-```
-
-## Instance Variables
-
-Instance variables are variables that are unique to each instance of a class. They can be set initially within the `__init__()` method when the instance is created, or instance variables can be added by setting them to values.
-
-```py
-class Dog:
-  def __init__(self, breed):
-    # Instance variable:
-    self.breed = breed
+class Home:
+  name="Code Ninja"
+  rooms = 4
+  stories = 2
+
+  @staticmethod
+  def is_on_market(home):
+    if(home.name == ""):
+      return True
+    else:
+      return False
+
+  @classmethod
+  def paint_wall(self, color):
+    return f"Painting wall with the color {color}."
 
-dog = Dog("Jindo")
+home = Home()
 
-dog.name = "Ruff"
+print(Home.is_on_market(home))
+# Output: False
 ```
 
-We give the instance `dog` a new instance variable called `name` and set it to the value `"Ruff"`.
-
-## **init**() Method
+#### Customized Class Methods
```

Comment:
It might be worth it to expand this to cover more dunder/magic methods.

Reply:
This actually reminds me of something I was considering. Should we do one of the following?

- Create a new `terms` folder under `concepts/classes` with entries for these methods. (e.g., `__init__.md`, `__repr__.md`, `__str__.md`)
- Create a new _concept entry_ (maybe it could be called `Dunder Methods`?) and then create new term entries under there?

Reply:
@Dusch4593, My vote would be for the latter. A new entry called "Dunder Methods".